invokeai/backend/prompting/conditioning.py

```diff
@@ -59,11 +59,15 @@ def get_uc_and_c_and_ec(prompt_string,
     if log_tokens or getattr(Globals, "log_tokenization", False):
         log_tokenization(positive_prompt, negative_prompt, tokenizer=model.tokenizer)
 
-    with InvokeAIDiffuserComponent.custom_attention_context(model.unet,
-                                                            extra_conditioning_info=None,
-                                                            step_count=-1):
-        c, options = compel.build_conditioning_tensor_for_prompt_object(positive_prompt)
-        uc, _ = compel.build_conditioning_tensor_for_prompt_object(negative_prompt)
+    # The below has been commented out as it is an instance method used for cleanly loading LoRA models, but is not currently needed. 
+    # TODO: Reimplement custom_attention for 3.0 support of LoRA.
+
+    # with InvokeAIDiffuserComponent.custom_attention_context(model.unet,
+    #                                                         extra_conditioning_info=None,
+    #                                                         step_count=-1):
+    #     c, options = compel.build_conditioning_tensor_for_prompt_object(positive_prompt)
+    #     uc, _ = compel.build_conditioning_tensor_for_prompt_object(negative_prompt)
+    #     [c, uc] = compel.pad_conditioning_tensors_to_same_length([c, uc])
```

Comment:
Are we sure this isn't used? I specifically had to add this line to make it stop crashing with long prompts (>75 tokens, see above convo). so clearly this code path was being taken. I hadn't had a chance to test it with this code commented.

Reply:
No... you just need delete context manager. 3 next lines generate conditioning tensors - code now simply do nothing.